src/nl/rubensten/texifyidea/action/preview/PreviewAction.kt

```diff
@@ -0,0 +1,70 @@
+package nl.rubensten.texifyidea.action.preview
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Key
+import com.intellij.openapi.wm.ToolWindowAnchor
+import com.intellij.openapi.wm.ToolWindowManager
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.psi.PsiElement
+import com.intellij.ui.content.ContentFactory
+import nl.rubensten.texifyidea.action.EditorAction
+import nl.rubensten.texifyidea.ui.EquationPreviewToolWindow
+import nl.rubensten.texifyidea.ui.PreviewFormUpdater
+import javax.swing.Icon
+
+abstract class PreviewAction(name: String?, val icon: Icon?) : EditorAction(name, icon) {
```

Comment:
I would appreciate it if you would add a comment to classes you added, what the class is about and why it exists. I know most existing classes don't have that, but it nevertheless helps me give some overview :)

Reply:
Good idea! I'll add this shortly, thanks for the review :)

Reply:
I've added some documentation. Please tell me if it is sufficient enough :)

Reply:
Yes, excellent! it will be very helpful in the future. I would wish everything in this codebase would be documented like this ;) 
@RubenSchellekens will also give this PR a good look later on, and as usual I will include this in alpha builds in the meantime (I think I'm going to release one today).